js/script.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+let content = document.getElementById("circle");
+let button = document.getElementById("button");
+
+button.onclick = function() {
+
+    if (content.className == "circle") {
```

Comment:
You can use here toggle active class mechanism. Read more about it and try to attach to the project.